frontend/.env

```diff
@@ -0,0 +1 @@
+VITE_MEASUREMENT_ID=G-SN8YCF7ES9
```

Comment:
I'm fairly certain that it's not. It can only be used to send user metrics for our application, so the worst someone could do is send false events. As it's used in the frontend I don't actually think we can keep it secret (As even if it was hidden on Github, it would be possible to extract it using some kind of Javascript injection or something)